app/jobs/order_cycle_webhook_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# Create a webhook payload for an order cycle, triggered by an event.
+# The payload will be delivered by another job.
+class OrderCycleWebhookJob < ApplicationJob
+  def perform(order_cycle_id, event)
+    order_cycle = OrderCycle.find(order_cycle_id)
+    data = order_cycle
```

Comment:
I'd call this variable webhook_payload so it's easier to see what it will be used for.